include/dynd/kernels/field_access_kernel.hpp

```diff
@@ -39,7 +71,7 @@ namespace ndt {
 
   template <>
   struct traits<nd::field_access_kernel> {
-    static type equivalent() { return type("({...}) -> void"); }
+    static type equivalent() { return type("({...}, field_name : string) -> Any"); }
   };
 
```

Comment:
A bit of bikesheding here, but what should this callable be called? Do people like `nd::field_access`? Ping @mwiebe @insertinterestingnamehere